PureLayout/PureLayout/PureLayoutDefines.h

```diff
@@ -34,10 +34,18 @@
 #define __PureLayout_MinBaseSDK_iOS_8_0                   (TARGET_OS_IPHONE && __IPHONE_OS_VERSION_MAX_ALLOWED > __IPHONE_7_1)
 #define __PureLayout_MinBaseSDK_OSX_10_10                 (!TARGET_OS_IPHONE && __MAC_OS_X_VERSION_MAX_ALLOWED > __MAC_10_9)
 
+#ifdef __MAC_10_11
+#define __PureLayout_MinBaseSDK_OSX_10_11                 (!TARGET_OS_IPHONE && __MAC_OS_X_VERSION_MAX_ALLOWED >= __MAC_10_11)
```

Comment:
I think there's a small problem with this line. The symbol `__MAC_10_11` may not be available on Mac OS 10.10, which we still support for backwards compatibility.

Did you happen to check your change against Mac OS 10.10?